src/jit/decomposelongs.cpp

```diff
@@ -454,6 +468,43 @@ GenTree* DecomposeLongs::DecomposeStoreLclVar(LIR::Use& use)
 }
 
 //------------------------------------------------------------------------
+// DecomposeStoreLclFld: Decompose GT_STORE_LCL_FLD.
+//
+// Arguments:
+//    use - the LIR::Use object for the def that needs to be decomposed.
+//
+// Return Value:
+//    The next node to process.
+//
```

Comment:
It would be useful to add an example input IR and output IR produced by this routine.